client/src/style/index.css

```diff
@@ -10,11 +10,79 @@ body {
   color: #37352B;
   font-size: 18px;
   height: 100vh;
-  background-color: #DCE8ED;
+  background-color: white;
 }
 
-h1, h2 {
+h1, h2, h3 {
   font-family: 'Saira Extra Condensed', sans-serif;
+  margin-top: 3%;
+  position: relative;
+}
+
+label {
+  font-weight: bold;
+  margin-top: 20px;
+  margin-bottom: 30px;
+  position: relative;
+}
+
+button {
+  font-size: 18px;
+  font-family: 'Source Sans Pro', sans-serif;
+  cursor: pointer;
+  background-color: white;
+  color: #04AA6D;
+  font-weight: 500;
+  padding: 10px 25px;
+  min-width: 250px;
+  text-align: center;
+  border: 1px solid #04AA6D;
+  border-radius: 7px;
+  position: relative;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+button:disabled,
+button[disabled]{
+  border: 1px solid #999999;
+  background-color: #cccccc;
+  color: #666666;
+  cursor: default;
+}
+
+textarea {
+  display: flex;
+  margin-bottom: 20px;
+  position: relative;
+  width: 50%;
+  box-sizing: border-box;
+  padding: 8px 10px;
+  border: 1px solid #454545;
+  border-radius: 4px;
+  font-family: 'Source Sans Pro', sans-serif;
+  color: #121212;
+  font-size: 16px;
+  min-height: 70px;
+  max-height: 300px;
+  background-color: #FEFEFD;
+}
+
+table {
+  border-collapse: collapse;
+  width: 85%;  
```

Comment:
you could update this to 100% if you want even spacing on either side of the tables

Reply:
thanks emi! will change it and merge